src/ec/packages/ec-component-expandable/expandable.html.twig

```diff
@@ -56,22 +56,14 @@
 {# Print the result #}
 
 <div class="{{ _css_class }}"{{ _extra_attributes|raw }}>
+  {% set _label = _label_expanded %}
   {% set _button_attributes = [
     { name: 'aria-controls', value: _aria_controls },
     { name: 'data-ecl-expandable-toggle', value: true },
     { name: 'data-ecl-label-expanded', value: _label_expanded },
-    { name: 'data-ecl-label-collapsed', value: _label_collapsed }
+    { name: 'data-ecl-label-collapsed', value: _label_collapsed },
+    { name: 'aria-expanded', value: _expanded }
```

Comment:
the aria-expanded attribute is handled by javascript in this component, so we just set it initially as false (at least this is the way it is done in ECL) and then is js playing with that. I see that by doing like you did we could offer the possibility of using the component with a "not-collapsed" state initially, but this is not what the ECL definition of the component does, so i would not introduce it here on our side as an extra parameter.

Reply:
Agree but this was already the case before the actual PR, I was also a bit confused with this, see the parameter here:
https://github.com/ec-europa/ecl-twig/blob/292c833c54b31c19edfdd97042fb16dc5cb405a5/src/ec/packages/ec-component-expandable/expandable.html.twig#L8

Anyway, I have removed the `expanded` parameter now.